PowerEditor/installer/nativeLang/german.xml

```diff
@@ -1531,6 +1533,10 @@ Sollen diese Dateil-Platzhalter erstellt werden?
 HINWEIS: Wenn die Platzhalter nicht erstellt oder später geschlossen werden, wird die Sitzung beim Beenden MODIFIZIERT! Es wird empfohlen jetzt eine Sicherung der Sitzung &quot;session.xml&quot; zu machen."/>
 			<RTLvsDirectWrite title="Kann nicht auf 'Schreibrichtung von rechts nach links' (RTL) schalten" message="'RTL' ist nicht kompatibel mit dem 'DirectWrite' Modus. Bitte dazu 'DirectWrite' unter 'Diverses' in den Einstellungen deaktivieren und danach Notepad++ neu starten."/>
 			<FileMemoryAllocationFailed title="Ausnahmefehler: Dateispeicherzuweisung fehlgeschlagen" message="Eventuell gibt es nicht genügend zusammenhängenden freien Speicher für die von Notepad++ geladene Datei."/><!-- HowToReproduce: Try to open multiple files with total size > ~700MB in the x86 Notepad++ (it will depend on the PC memory configuration and the current system memory usage...). -->
+			<FindRegexBackwardDisabled title="Regex Rückwärtssuche nicht aktiv" message="Standardmäßig ist die Regex Rückwärtssuche deaktiviert da möglicherweise unerwartete Ergebnisse auftreten. Um doch eine Rückwärtssuche durchzuführen ist in der Suche entweder die normale oder erweiterte Suche statt regulärer Ausdrucke auszuwählen.
+OK drücken um die Suche zu öffnen oder in den Vordergrund zu bringen.
+			
+Wenn die Regex Rückwärtssuche doch benötigt wird bitte in der Anleitung dafür nachlesen."/>
```

Comment:
Vorschläge:
Regex-Rückwärtssuche mit Bindestrich

... "Um dennoch eine Rückwärstssuche durchzuführen, ist in der " ...

anstatt "OK drücken", besser "OK klicken" oder "OK auswählen" ?

letzter Satz:
"Falls die Regex-Rückwärtssuche dennoch benötigt wird, kann deren Aktivierung im Benutzerhandbuch nachgeschlagen werden."